### PR TITLE
pkgs/os-specific: stdenv.lib -> lib

### PR DESCRIPTION
**Pull request by siraben**
###### Motivation for this change
Pending merge of #109455

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by jonringer** [comment]
this shouldn't be causing any rebuilds...

**Comment by siraben** [comment]
@jonringer The reason why might be because of whitespace changes.

**Comment by jonringer** [comment]
ah that makes sense, and it's unfortunate.

If saw `rebuilds: 0`, then it makes it easier to merge as there's almost no risk in breaking something.... and the linting gate can just be bypassed.

File: pkgs/os-specific/linux/kernel/common-config.nix [hunk 1]
```diff
@@ -10,14 +10,14 @@
 # hardware problems with a new one.
 
 # Configuration
-{ stdenv, version
+{ lib, stdenv, version
```
Comment (siraben) [inline pkgs/os-specific/linux/kernel/common-config.nix:13]: @jonringer This was a change I had to do manually to fix the evaluation error at some point.

**Comment by prusnak** [comment]
> @jonringer The reason why might be because of whitespace changes.

Seems there is a whitespace change here: https://github.com/NixOS/nixpkgs/pull/109490/files#diff-0e2f4255d92df895a1e001d696295864233cb319f2aa78aa10654efef9ef1663L33-R33

**Comment by siraben** [comment]
@prusnak but that whitespace change is in the `longDescription` which shouldn't affect the hash.

**Comment by prusnak** [comment]
OK, TIL. There is another whitespaces change in `pkgs/os-specific/darwin/apple-source-releases/text_cmds/default.nix` in `patchPhase`. Maybe that explains why there is no rebuild on linux, but there is on darwin.

**Comment by siraben** [comment]
@prusnak Thanks, that made rebuilds go to 0.

@SuperSandro2000 pending one last ofborg CI check.

**Pull request merged**